src/common/utils/useBackendSorting.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+
+import { usePrevious } from './usePrevious';
+
+interface SortedCol {
+  id: string;
+  desc?: boolean;
```

Comment:
Would it make sense to create enum `SortDirection` with values `ASC` and `DESC` ?

Reply:
I defined here the same interface used in `react-table` to avoid transformations.